flask_project/url_scanner.py

```diff
@@ -0,0 +1,33 @@
+"""
+Url scanner module to work with urls/domains/ips.
+"""
+import json
+import requests
+import urllib
+from urllib.parse import urlparse
+
+from config import DevelopmentConfig
+
+
+class IPQS:
+    """
+    IP Quality Score class used to send request to IPQualityScore API and get results about url.
+    """
+
+    key = DevelopmentConfig.IPQS_SECRET_KEY  # API SECRET KEY
+
+    def malicious_url_scanner_api(self, url: str, vars: dict = {}) -> dict:
+        url = "https://www.ipqualityscore.com/api/json/url/%s/%s" % (
```

Comment:
Please store it as a global variable in some configuration file, e.g. config.py

Reply:
You mean url or key? Because DevelopmentConfig.IPQS_SECRET_KEY is from config.py

Reply:
urls should be configurable as well to avoid changes in the source code in case of the url change.

Reply:
urls should be configurable as well to avoid changes in the source code in case of the url change .

Reply:
urls should be configurable as well to avoid changes in the source code in case of the url change.